sdk/identity/identity/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Breaking Changes from 2.0.0-beta.4
 
 - Removed the `allowMultiTenantAuthentication` option from all of the credentials. Multi-tenant authentication is now enabled by default. On Node.js, it can be disabled with the `AZURE_IDENTITY_DISABLE_MULTITENANTAUTH` environment variable.
+- Removed rdded regional support from client credential types. Also removed support for the `AZURE_REGIONAL_AUTHORITY_NAME` environment variable.
 
```

Comment:
nit: rdded -> added. Also, I think this should mention that we intend to add these back into the next beta release. Finally in .NET we didn't remove support for the environment variable, but we also didn't mention explicitly that it was still supported. This was intended to give first party applications which are going to be required to use regional STS a way to use it with the Azure SDK without adding an API that third party apps might stumble upon.

Reply:
What do you think of the new entry?

Reply:
Looks great.